MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -433,10 +501,27 @@ - (void)acquireTokenInteractive:(id)sender
                 [self updateResultViewError:error];
             }
             
+            [_webView loadHTMLString:@"<html><head></head><body>done!</body></html>" baseURL:nil];
+            [_authView setHidden:YES];
+            [self.view setNeedsDisplay];
+            
             [[NSNotificationCenter defaultCenter] postNotificationName:MSALTestAppCacheChangeNotification object:self];
         });
     };
     
+    application.webviewSelection = _webviewSelection.selectedSegmentIndex == 0 ? MSALWebviewSelectionEmbedded : MSALWebviewSelectionSystemDefault;
+    
+    if (application.webviewSelection == MSALWebviewSelectionEmbedded &&
+        [[_customWebViewSelection titleForSegmentAtIndex:_customWebViewSelection.selectedSegmentIndex] isEqualToString:TEST_EMBEDDED_CUSTOM])
```

Comment:
not sure why we are checking the title here? Can't we just assign an option to each segment?

Reply:
It was done purely for readability purposes.